client/app.js

```diff
@@ -13,6 +14,7 @@ window.sprintf = require('sprintf-js').sprintf
 window.c3 = require('c3/c3.js')
 window.d3 = require('d3/d3.js')
 window.patternflyVersion = 4
+window['numeral'] = numeral
```

Comment:
thinkin' this oughta follow the pattern from lines 15 and 14... just cuz... ya know, keeping things consistent

soooo...

`window.numeral = require('numeral')` (especially considering cuz we're bringing in the whole lib)